boards/BMS/Test/Src/Test_StateMachine.cpp

```diff
@@ -279,30 +276,26 @@ class BmsStateMachineTest : public BaseStateMachineTest
     struct Clock *            clock;
 };
 
-// BMS-31
 TEST_F(BmsStateMachineTest, check_init_state_is_broadcasted_over_can)
 {
     SetInitialState(App_GetInitState());
     EXPECT_EQ(BMS_INIT_STATE, App_CanTx_BMS_Vitals_CurrentState_Get());
 }
```

Comment:
We no longer track requirements so removed all the comments from Test_StateMachine.cpp